proposals/simplified-td/index.html

```diff
@@ -612,19 +635,65 @@ <h3><code>@id</code></h3>
 
       <section>
         <h2>Simplified TD by Example</h2>
-
+        
         <p>
-          The first example reflects the most progressive option with the
-          highest amount of preprocessing necessary. It contains no semantic
-          annotations, nor human-readible decorators such as <code>name</code>
-          or <code>description</code>.
+          When no <code>@context</code> is given, the <a href="https://w3c.github.io/wot/w3c-wot-td-context.jsonld">TD context file</a>
+          is assumed based on the media type. The current TD context needs an additional
+          definition of <code>@base</code> set to the Thing's <code>@id</code>,
+          so that the object representation of Interactions (instead of arrays)
+          works correctly.
         </p>
 
-        <pre class="example" title="Progressive Simplified TD Example">
+        <pre class="example" title="Context applied to application/td+json">
+{
+  "@context": {
+    /* valid @id generation relative to Thing instance */
+    "@base": "@id",
+    /* object notation for Interactions */
+    "properties": {
+      "@container": "@id"
+    },
+    "actions": {
+      "@container": "@id"
+    },
+    "events": {
+      "@container": "@id"
+    },
+    /* optional if schema definitions are wanted */
+    "definitions": {
+      "@container": "@id"
+    },
+    /* TD term definitions as in https://w3c.github.io/wot/w3c-wot-td-context.jsonld */
+    ...
+  }
+}
+        </pre>
+        
+        <section>
+          <h3>Minimal</h3>
+
+          <p>
+            The first example reflects the a minimal option. It contains no semantic
+            annotations, nor human-readible decorators such as <code>name</code>
+            or <code>description</code>.
+          </p>
+          <p>
+            Update 2018-03-06: Applied JSON-LD features for Data Schema definitions.
+          </p>
+
+          <pre class="example" title="Simplified TD Example">
 {
   "name": "lamp",
   "@id": "urn:dev:wot:com:example:servient:lamp",
   "base": "https://servient.example.com/",
+  /* optional, only if wanted, cf. Dave's proposal! for free by JSON-LD */
+  "definitions": {
+    "range": {
```

Comment:
maybe the name could be less generic, e.g. range_0_100

Reply:
It is up to the TD author how they want to name their custom types in the TD. It is just an optimization to not repeat definitions (=schemas).

When people start to define "public types" that would be imported via an external context (e.g., domain-specific .jsonld file given), they should put such thoughts into the term design.